packages/core/src/Modules/Cashier/Components/cashier-container.jsx

```diff
@@ -2,27 +2,24 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import Loading from '../../../templates/_common/components/loading.jsx';
 
-class CashierContainer extends React.Component {
-    render() {
-        return (
-            <div className='cashier__wrapper'>
-                {this.props.is_loading && <Loading />}
-                {this.props.iframe_url && (
-                    <iframe
-                        className='cashier__content'
-                        height={this.props.iframe_height}
-                        src={this.props.iframe_url}
-                        frameBorder='0'
-                        scrolling='auto'
-                    />
-                )}
-            </div>
-        );
-    }
-}
+const CashierContainer = ({ iframe_height, iframe_url, is_loading }) => {
```

Comment:
Just a nitpick: Can we omit brackets and return statement since there is no condition present: `is_loading }) => (<div`

Reply:
@akmal-binary  Actually, I believe omitting brackets and return will decrease the code readability in this situation, what do you think?

Reply:
Fair enough, for me it actually looks quite the same, just shorter, but we can keep the current version as well